modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+If a {productname} instance had a custom notification, `+editor.notificationManager.open+`, set to display when the *Fullscreen* button was used, toggling from Fullscreen back to default resulted in the notification displaying outside the {productname} editor.
+
+{productname} 6.1 corrects this. When toggling between the two *Fullscreen* states, custom notifications now display immediately below the toolbar in either state, as expected.
+
+
+=== Dialogs no longer exceed window height on smaller screens
+
+When re-sizing the viewport while a {productname} dialog is open, the dialog did not start re-sizing until the viewport boundary begins encroaching on the dialog. As a consequence, the dialog was partially cut off in the re-sized viewport (specifically, the bottom border and several pixels of the dialog’s buttons were over-shot) and it could become difficult to distinguish the dialog from the viewport.
+
+With this update, this behavior has been corrected. When a {productname} dialog is open and the viewport is re-sized, the dialog now re-sizes such that both the top and bottom borders of the dialog are always within the current viewport.
+
+
+=== Some function types in the TreeWalker API missed that it could return `undefined`
+
+Backspacing through material in an active {productname} editor could, in some circumstances, result in the insertion point stepping into a table and continue deleting where the expected behavior was for the insertion point to stop on the line below the table.
+
+In investigating and fixing this issue (see immediately below) it was also found that some function types in the `tinymce.dom.TreeWalker` class did not detect that the class could return an `undefined` value.
+
+This has been corrected and all `tinymce.dom.TreeWalker` functions now correctly detect `undefined` values when these are returned.
+
+
+=== In some cases pressing the Backspace or Delete key would incorrectly step into tables rather than remain outside
+
+In some circumstances, when a table was followed immediately by a text paragraph, and the user deleted the text paragraph by back-spacing through each character, pressing the backspace character after the last character was removed resulted in the insertion point moving in to the table. If the user continued pressing the backspace key, material in the table’s bottom-right cell was deleted.
+
+The expected behavior when backspacing on an empty line immediately following a table is for the insertion point to not move at all.
+
+With this update, {productname} behaves as expected. Pressing the backspace key on an empty line immediately following a table produces no change to the insertion point location.
+
+
+=== Text alignments, such as flush left and centered, could not be applied to `<pre>` elements
```

Comment:
Is `flush` correct here? I've never heard that mentioned and seems out of context, so it might be confusing to customers reading this as well.

Reply:
[Flush left, and flush right](https://en.wikipedia.org/wiki/Typographic_alignment), are the standard terms for these two typesetting options.

The other two in the standard set of four are justified (aka fully-justified) and centred.